src/App.js

```diff
@@ -65,9 +67,14 @@ const App = compose(
             forceSelectRole({
               api,
               isLoadingUser,
-              Component: ({ match, ...props }) => (
-                <ArrangerDashboard basename={match.url} {...props} />
-              ),
+              Component: ({ match, ...props }) => {
+                if (
+                  !isAdminToken({ validatedContent: validateJWT({ jwt: state.loggedInUserToken }) })
+                ) {
+                  history.replace('/dashboard');
+                }
+                return <ArrangerDashboard basename={match.url} {...props} />;
```

Comment:
similar to forceSelectRole, could we use a `<Redirect to="..." />` rather than modifying url directly?
```
return !get(...).includes('ADMIN') ? <Redirect to="..." /> : <ArrangerDashboard ... />
```